frontend/src/app/store/actions/dashboard.actions.ts

```diff
@@ -62,4 +63,27 @@ export const openSidebar = createAction(
 
 export const closeSidebar = createAction(
     '[Dashboard page] Close sidebar'
+);
+
+export const markTaskCompleted = createAction(
```

Comment:
It is better to use common angular/ngrx naming rules. Specificly you would have: mark comleted, mark completed success, mark completed failed. Similar issues with '[API] Task marked completed' - api is not really correct.
Take a look with example in these articles, some may be helpful. Find more materials on your own.
https://github.com/orizens/ngrx-styleguide/blob/master/docs/actions.md
https://christianlydemann.com/ngrx-industry-best-practices/
https://indepth.dev/posts/1451/ngrx-best-practices-new

Reply:
Fixed